interceptors/src/main/java/uk/gov/moj/sdt/interceptors/in/SdtUnmarshallInterceptor.java

```diff
@@ -70,6 +70,7 @@ public SdtUnmarshallInterceptor() {
     public void handleMessage(final SoapMessage message) throws Fault {
         // Call the following interceptors in the chain.
         final PhaseInterceptorChain interceptorChain = (PhaseInterceptorChain) message.getInterceptorChain();
+        LOGGER.debug("SoapMessage: {}", message);
```

Comment:
Is this debug message something that was used during development?  If so it may be able to be removed.